# Traduction transcript

In [1]:
from youtube_transcript_api import YouTubeTranscriptApi
from deep_translator import GoogleTranslator
import unicodedata
import csv
import chardet
from datetime import datetime
import time
import glob
import string
import pandas as pd
from pytube import YouTube
import nltk
from detect_delimiter import detect
from mtranslate import translate
import re
from langdetect import detect
from itertools import zip_longest

In [4]:
characters = "/_\[]@|~{}@!?"
def supp_special_carac (srt):
  all_trans=""
  for i in srt:
    
    if ((i['text'].lower() == "[musique]")or (i['text'].lower() == "[music]")) :
      i['text'] = " "
                
    if ((i['text'].lower() == "[applause]") or (i['text'].lower() == "[applaudissements]")):
      i['text'] = ""
    for x in range(len(characters)):
      i["text"] = i["text"] .replace(characters[x],"")
    a=re.sub("\d\d\:\d\d|\d\d\\s:\s\d\d|\d\d\\s:\d\d|\d\d\:\s\d\d","",i["text"])
    
    all_trans= all_trans+" "+ a
    
  
  return(all_trans)

In [7]:

file = open('liens2.txt', "r")
#i pour les transcription des video , j : avec traduction , les cas particilier : compteurs pour chaque cas d'utilisatiion 
l=0
j=0
k=0
lines = file.readlines()

with open('sous_titrage.csv',  "w",encoding='utf-8-sig') as f:
  fieldnames = ['lien_youtube','sous_titrage']
  writer = csv.DictWriter(f, fieldnames=fieldnames)
  writer.writeheader()
  for i ,line in  enumerate (lines):
    print("document num :" , i)
    yt = YouTube(line)
    srt = YouTubeTranscriptApi.get_transcript(str(yt.video_id) ,languages= ['fr', 'en','en','en-CA','en-GB','fr-CA'])
    txt= supp_special_carac(srt)
    print(detect(txt))
    if (detect(txt)== 'fr') :
        l=l+1
        
        txt = unicodedata.normalize('NFKD', txt).encode('ascii', 'ignore') 
      
        writer.writerow({'lien_youtube':str(line),'sous_titrage':(txt)}) #video avec transcription  français
        continue
    else :
        try:
            translated = translate(txt, 'fr', "auto")
            j=j+1
            #print("##################",translated)
            writer.writerow({'lien_youtube':str(line),'sous_titrage':(translated) })#video n'a pas de transcription  français avec lengeur de texte <1000
  
        except:
               with open("lien5.txt", "a") as f0:
                f0.write("\t{}".format(line))
                k=k+1
#continue
       
     

document num : 0
fr
document num : 1
fr
document num : 2
fr
document num : 3
fr
document num : 4
fr
document num : 5
fr
document num : 6
en
document num : 7
fr
document num : 8
fr
document num : 9
fr
document num : 10
en
document num : 11
fr
document num : 12
fr
document num : 13
fr
document num : 14
fr
document num : 15
fr
document num : 16
fr
document num : 17
fr
document num : 18
fr
document num : 19
fr
document num : 20
en
document num : 21
fr
document num : 22
fr
document num : 23
fr
document num : 24
fr
document num : 25
en
document num : 26
fr
document num : 27
fr
document num : 28
en
document num : 29
fr
document num : 30
en
document num : 31
fr
document num : 32
en
document num : 33
fr
document num : 34
fr
document num : 35
fr
document num : 36
en
document num : 37
en
document num : 38
fr
document num : 39
en
document num : 40
en
document num : 41
fr
document num : 42
fr
document num : 43
fr
document num : 44
fr
document num : 45
fr
document num : 46
fr
document num : 47
fr
do

In [12]:
print(l,j,k)

133 18 25


on a 133 videos qui ont des transcriptions en francais,18 traduite par le translator , 25 qu'on n'arrive pas à les traduire 


In [13]:
#enlever les emotions , symboles et flags
emoji_pattern = re.compile("["
        u"\U0001F600-\U0001F64F"  # emoticons
        u"\U0001F300-\U0001F5FF"  # symbols & pictographs
        u"\U0001F680-\U0001F6FF"  # transport & map symbols
        u"\U0001F1E0-\U0001F1FF"  # flags (iOS)
                           "]+", flags=re.UNICODE)
#print(emoji_pattern.sub(r'', text)) # no emoji

In [14]:
file = open('lien5.txt', "r")
lines = file.readlines()
print(len(lines))

25


In [15]:


f= 0
w = 0
with open('sous_titrage.csv',  "a",encoding='utf-8-sig') as f:
  fieldnames = ['lien_youtube','sous_titrage']
  writer = csv.DictWriter(f, fieldnames=fieldnames)  
  for i ,line in  enumerate (lines):
        trans = " "
        print("document num :" , i)
        print(line)
        yt = YouTube(line)

        
        srt = YouTubeTranscriptApi.get_transcript(str(yt.video_id),languages=('en','en-CA','en-GB','fr-CA',))
        
           
        txt= supp_special_carac(srt)
        txt = unicodedata.normalize('NFKD', txt).encode('ascii', 'ignore') 
            #print(txt)
        for i in srt :
            w=w+1
            translated = translate(emoji_pattern.sub(r'', i["text"]), 'fr', "auto")
            trans= trans+" "+translated
        print(trans)                
        writer.writerow({'lien_youtube':str(line),'sous_titrage':str(trans)})#video n'a pas de transcription  français avec lengeur de texte >1000
       

                
                
            
  
      
 


document num : 0
	https://www.youtube.com/watch?v=JysGq191aso

  Peter Piper a choisi un morceau de mariné
les poivrons cherchez-vous vraiment à économiser quelques graines de poivre dans cette vidéo je suis
va vous montrer à quel point c'est simple et facile est de conserver vos graines de poivre Hé les gars, je voulais vous montrer exactement comment
pour savoir quand tu vas choisir ton plantes de poivre maintenant vous voulez vous assurer
que vos poivriers sont bien mûrs et pleinement mature signifie qu'ils ont
terminé leur cycle de croissance ou très proche de la fin et du chemin
que je le sais, le moyen le plus simple est que si tu connais le poivre de quelle couleur
ça va être quand c'est seulement mature alors c'est généralement le moment idéal où il
arrive à cette couleur particulière que vous connaissez qu'il a atteint sa pleine maturité
parce que parfois tu choisiras bien la plupart les gens vont cueillir les poivrons quand
ils sont plus verts et ils ne sont pas entièrement m

  On dirait que j'ai du cassé là-dedans.
on dirait que j'ai beaucoup de cassé là-dedans. musique d'introduction Eh bien, nous sommes enfin sur notre chemin pour aller essayer
et couper du grain. Allez voir si c'est assez sec. Nous sommes plus tard
que nous pensions que nous serions, Eh bien, alors nous aimerions être, mais nous ne pouvons pas
aide le. Le grain prend juste ça longtemps pour se préparer à partir, mais.
Je suis dans la moissonneuse-batteuse qui suit Phillip. Il a la tête.
Là-haut devant moi, tourne juste sur la route.
Nous allons nous diriger vers ça petit morceau que nous avons. C'est en fait
le seul blé d'hiver que nous ayons. Il a une petite retouche dessus,
du blé de printemps, mais descendez là-bas l'en-tête, coupez, voyez à quel point c'est sec.
Voir si nous pouvons allez-y.
Nous avons coupé les extrémités et coupé un peu. L'humidité semble être un peu élevée. Désolé je cherche mon testeur.
Nous avons l'accoudoir mais nous avons eu ceci toujours.
Ceci est notre prop

  qu'est-ce que l'écureuil a le goût de l'écureuil a le goût de l'écureuil depuis un an je n'ai J'ai mangé des aliments que j'ai cultivés et en ai cherché un année sans épiceries ou restaurants rien emballé ou traité rien expédié sur de longues distances littéralement tout ce que j'ai toute l'année vient de mon jardin ou que j'ai récolté de la nature mon nom est Rob Greenfield Je suis un militant écologiste qui se lance dans des aventures extrêmes pour apporter attention aux importants aspects environnementaux et questions sociales essentiellement, je crois que notre alimentation industrielle mondiale actuelle le système est cassé partout où vous regardez cela cause de graves destructions à les gens aux autres espèces et à la terre dans son ensemble et donc je voulais faire quelque chose d'extrême, j'ai toujours eu le question serait-il possible d'avancer loin de la grande AG s'éloigner de la système alimentaire industriel mondial et pourrait J'existe sans épiceries et restaurants et c

  Arbres de prise de force. Quels sont
ils, comment fonctionnent-ils, et comment trouvez-vous l'arbre qui est
bon pour vous Salut tout le monde, ici Jim. Et dans cette vidéo, Je vais tout vous dire sur le pouvoir
Take Off Shafts, leurs composants de base, trucs et astuces sur
comment identifier le fabricant, séries et tailles; Et je vais te montrer
comment mesurer... le centre fermé des arbres. Allons-y. Les arbres de prise de force, ou arbres de prise de force, sont largement utilisés dans l'industrie agricole. Nous les voyons dans les cultivateurs, les presses à balles, les faucheuses, machines de récolte, et bien d'autres
applications. Ils transfèrent le pouvoir d'un pouvoir
source à l'équipement. Par exemple, du tracteur à l'outil. Lorsqu'un arbre de prise de force est la connexion principale
entre le tracteur et l'équipement... nous l'appelons l'arbre primaire. S'il est utilisé ailleurs sur
la machine, par exemple, pour connecter... deux boîtes de vitesses à une tondeuse, nous app

   Salut. Planter des graines à la main est important depuis plus de dix mille ans depuis que l'homme s'est impliqué dans l'agriculture et il existe quelques types de planteurs manuels de base qui ont été utilisés. des choses comme les jardinières de type jab. Maintenant, ces planteurs de type jab fonctionnent assez bien pour les grosses graines comme le maïs ou les haricots Mais ils ne fonctionnent pas bien pour les petites graines. Maintenant, quand je dis de petites graines, je veux dire des graines comme des graines d'alyssum sucrées Je fais des recherches sur l'alyssum sucré depuis plusieurs années car c'est un plante très importante que nous plantons dans nos systèmes de légumes en Californie pour attirer chez les insectes bénéfiques. Des insectes comme le syrphe. Les mouches volantes sont vraiment importantes car elles nous aident à contrôler les parasites comme les pucerons et les syrphes ont besoin de pollen et de nectar de fleurs d'alyssum ou d'autres fleurs pour aider nous f

    Lors de la plantation de parcelles alimentaires ou indigènes
l'herbe de saison chaude dans la forêt représente la faune, cela peut être très bénéfique
à l'aide du semoir sans labour. Cela garantit que vous aurez un bon sol de semis
contact, profondeur de plantation uniforme et conservation de l'humidité du sol et
matière organique. Cela conduira à un bon succès de plantation
et peut réduire le risque d'échec. Lors de l'utilisation d'un semoir sans labour,
ou toute méthode de plantation, il est très important de calibrer
votre matériel de semis. Souvent, en raison de la taille variable des graines et
quantités de paille et graines vivantes pures dans un mélange de graines. Le taux de semis réel peut varier du
réglage recommandé sur votre semoir sans labour. Grâce au calibrage, nous pouvons ajuster la quantité exacte de pur
semences vivantes plantées par acre. Cela garantit que nous plantons à
le taux recommandé par acre. Et ne manquera pas de semence,
ensemencer trop légèrement, ou 

  nous avons les grands en-têtes maintenant et nous sommes
couper l'orge c'est très excitant et c'est va être ma première rangée d'orge et mon
première fois avec les grosses têtes cette année donc je vais vous montrer à quoi ressemblent les grands en-têtes
comme dans une seconde après que je sois entré dans ma rangée mon en-tête est d'environ 30 pieds de long et l'autre
ceux sont 45 nous n'avons pu commencer qu'à c'est après le déjeuner parce que la région où nous cultivons était
censé recevoir une énorme tempête de grêle la nuit dernière mais ils ont juste un peu nous avons juste un
un peu à la fin de cette rotation de dumping en ce moment je viens d'allumer ma moissonneuse-batteuse et ça
est le grand en-tête donc nous allons commencer l'orge en ce moment c'est un peu nouveau au début
parce que c'est le défi, mais vous venez de il faut faire attention l'orge c'est comme du beurre à couper
c'est absolument incroyable et ça va si bien mais c'est absolument le plus irritant si vous devez

  bienvenue dans la suite de l'histoire
comment va tout le monde nous avons été en quelque sorte Eh bien, pas du tout, nous avons été assez occupés
ici dernièrement, comme je l'ai dit dans le passé, je je ne sais pas ça fait quelques mois
probablement mais j'ai dit que quand il s'agit il est temps de mettre de l'avoine ou de commencer à planter du maïs
et les haricots, il se tourne généralement vers plus une sorte de panique de dernière minute
chose où c'est la décision de commencer la plantation se fait généralement la nuit
avant généralement sans aucune idée que nous allaient commencer c'est exactement ce que
arrivé quand est venu le temps de mettre de l'avoine dans le sol donc ce que vous voyez nous
stocké notre avoine dans le sous-sol et à l'extérieur du temps et où il n'y a pas d'humidité
pourrait y arriver mm-hmm excusez-moi et fondamentalement où est hors du chemin maintenant
que nous avons bien arraché notre avoine à au moins, allons-y et disons que nous sommes
déposer le sur c

  bonjour et bienvenue à la récolte de paille fs19
addon et aujourd'hui on va faire un court petit tuto sur l'utilisation du Krone
premos 5000 je suis ukgamer808 et je vais vous guider à travers cela
tutoriel pour l'addon de récolte de paille fs19 qui est sorti l'autre jour nous sommes
faire ça sur ps4 alors qu'est-ce que c'est tout sur
eh bien, tout tourne autour de cette machine ici c'est le KRONE PREMOS 5000 c'est
l'essentiel de la récolte de paille addon ou DLC de récolte de paille pour l'agriculture
simulateur et quel morceau de kit craquant c'est maintenant c'est assez cher c'est
pas bon marché, aucun de ces kits n'est bon marché et c'est assez haut sur la peinture de la fente comme
bien où il est -39 sur la fente et chacun que vous avez après cela n'est que -
- mais ça ne sert à rien d'avoir des charges sauf si vous êtes en multijoueur car
vous ne pouvez pas utiliser un travailleur, vous devez le faire manuellement alors qu'est-ce que ça coûte
eh bien ça coûte un peu cher argent

  Salut tout le monde, bienvenue dans le
chaîne si vous êtes nouveau cet automne nous, dans les fermes brunes, avons eu l'occasion de
démo ce glaneur s97 donc nous l'avons couru pour trois à peu près trois jours complets et des haricots
en gros un jour et demi dans le maïs mais j'y vais mettre quelques acres ensemble mettre quelques heures dessus
nous avons eu beaucoup de questions sur ce que nous en pensions exécute actuellement deux lexi sur 740, donc c'est ce que
nous venons de papa, il a grandi en courant des nettoyeurs donc c'était la première fois qu'il en faisait partie depuis
un n5 donc j'ai pensé que nous ferions une discussion rapide à propos de ce que nous aimons, de ce que nous n'aimons pas et des choses que
nous a surpris et euh vous connaissez notre avis global là-dessus d'accord eh bien nous commençons ici nous allons
allez-y et ouvrez le panneau pour voir voir ce que nous avons ici ouais ça le fait et puis ça monte oh ouais ouais on y va donc y'a pas un tout
beaucoup à 

  Cette semaine sur Machines Pete
TV, tout, de la récolte à la livraison est couverte par le
équipement sur cette vente aux enchères de l'Ohio, et ce 7230 super propre
la moissonneuse-batteuse avait des enchérisseurs leurs mains. Toute une vie Deere
le propriétaire est particulièrement fier de cet immaculé 60. Et un
appel téléphonique du taxi enregistré le jour pendant la récolte.
VOTRE MACHINE EST UN SÉRIEUX INVESTISSEMENT – ET AU CŒUR
DE CHAQUE OPÉRATION AGRICOLE. QUELQUES APPELEZ ÇA UNE PASSION. ÉTAIENT
MACHINES PETE TV, ET AUJOURD'HUI NOUS COUVRIRONS TOUT
DES TOURS D'ENCHÈRES AUX ENCHÈRES DES CLASSIQUES, AUX DERNIÈRES TENDANCES
ET LA TECHNOLOGIE. MACHINES PETE – LE NOM LE PLUS FIABLE DE LA FERME
ÉQUIPEMENT. Machinerie Pete merci ces premiers sponsors pour leur
soutien - SULLIVAN AUCTEURS. LAISSEZ NOTRE ÉQUIPE DE PROFESSIONNELS
VOUS MONTRER COMMENT FAIRE VOTRE VENTE AUX ENCHÈRES RÉUSSIE. VISITE
SULLIVAN AUCTIONEERS.COM. Hé, les gens, bienvenue à Machinery Pete
LA TÉLÉ. Venir à vous 

  Avez-vous envisagé le travail du sol en bandes sur
votre ferme Cette série vidéo en six parties est va regarder l'équipement, l'économie
et comment faire travailler le travail du sol en bandes pour vous. Suivez-nous pendant que nous discutons des avantages, des
les inconvénients et les défis du strip-till systèmes avec quatre agriculteurs ontariens qui
ont leur propre approche unique du strip-till. Dans cette première vidéo, nous aborderons
les bases de ce qu'est un système de strip-till. Je suis Ken Nixon. Notre ferme est Ain Lee Farms.
Nous sommes situés entre Ilderton et Arva, et je cultive avec mon frère
Kevin et mon père George. Parlez-moi un peu du fonctionnement de la fraise à bande,
qu'est-ce que ça fait aux résidus et qu'est-ce que ça fait faire au sol Donc, notre plate-forme de strip-till qui est
un Dawn Pluribus a essentiellement un soc en plomb qui tranche essentiellement les résidus et crée un
une petite fissure, derrière cela se trouve un ensemble de rangées nettoyants 

  Le silence >> Bruit de fond Bienvenue
et merci d'être là. Tous les participants seront dans un
mode écoute seule jusqu'à la question et répondre à la conférence téléphonique d'aujourd'hui. À ce moment-là, veuillez appuyer sur l'étoile 1
votre téléphone à touches pour poser votre question. Veuillez réactiver votre ligne et indiquer votre
nom clairement afin que je puisse vous annoncer. L'appel d'aujourd'hui est enregistré. Si quelqu'un a des objections
vous pouvez vous déconnecter à ce moment. Je voudrais vous présenter votre hôte
pour l'appel d'aujourd'hui, M. David Lamm. Monsieur, vous pouvez commencer. >> David Lamm : Merci. Et bienvenue mesdames et messieurs jusqu'à la fin dans le bassin du fleuve Mississippi
Série de webinaires sur l'Initiative des bassins versants. Et le sujet d'aujourd'hui est la gestion des cultures de couverture pour
Gestion des éléments nutritifs : le point de vue d'un agriculteur et vous êtes vraiment pour un régal. Je pense que le présentateur d'aujourd'hu

  Salut, je suis Daniel Smith, régional du sud-ouest
agronome de l'université Wisconsin et UW Extension Nutriment Ravageur
programme de gestion et aujourd'hui je vais pour vous parler de couvert intercalaire
alors envisagez d'intercaler des cultures de couverture. Vous pouvez accomplir cette tâche en plusieurs
différentes méthodes. L'une consisterait à diffuser planter. Ainsi, sur ces photos, vous pouvez voir un
épandeur de diffusion à l'arrière d'un tracteur. Vous pouvez les récupérer pour moins
mille dollars parfois même moins Cinq cent dollars. Et ils font une belle
travail de semence et d'engrais à la volée et cultures et chaux à travers le paysage, et
ils peuvent être un bon outil à avoir dans le boîte à outils pour certaines cultures de couverture
applications. Et vous pouvez voir dans ce photo ceux-ci ont été diffusés dans l'avoine
maïs de semence un beau résidu lourd de résidu là-bas que nous pourrions facilement
semis direct dans. Nous pensons à interseeding, nous recevons sou

  bonjour tout le monde c'est un vrai honneur être invité à présenter mes informations aujourd'hui du Rodale Institute et moi excusez-moi de ne pas pouvoir le faire en français et je vais vous mettre tous au défi de écoute en anglais et je vais supposons que ce que josee a dit est vrai mais Je ne sais pas si je veux parler d'un quelques choses aujourd'hui je vais concentrer mon attention et mon énergie sur l'histoire de nos systèmes agricoles à long terme essai, je vais l'utiliser comme base parler de mais je vais parler à propos d'un tas de choses différentes ce est une photographie de notre fondateur de la Rodale Institute son nom est Jay I rodel ce n'était pas un agriculteur c'était un homme d'affaires mais il a compris l'importance de la nourriture c'est qu'il mangeait tout seul santé donc ce qu'il nous disait comme agriculteurs quand il a écrit cette phrase sur un tableau noir en 1942 un sol sain équivaut à une alimentation saine équivaut à des personnes en bonne santé quoi il nou

  On y va… un peu plus… un peu plus… c'est un peu beaucoup là quand même… on marche un peu plus, c'est une grosse surprise 3, 2 , 1: Bienvenue chez Agri Inside Agri Inside est un road movie, à travers la France, qui parlera d'agriculture, vignes, sylviculture et élevage, l'occasion de découvrir des machines incroyables et des rencontres mémorables avec des amoureux de la terre. Savez-vous que nous passons un moment incroyable Oui on dépasse les limites de tout.. c'est clair Nous voici à Bois les Pargny, nous allons découvrir deux ensembles de machines incroyables, en termes de puissance et de taille. C'est le semis de betteraves Après les semis de betteraves, nous allons vivre une révolution dans l'agriculture, une nouvelle page se tourne, certainement Allons-nous commencer Donc tout fonctionne avec le GPS, n'est-ce pas Kevin Oui Donc vous ne faites rien, vous devez surveiller l'outil de toute façon Oui nous surveillons l'outil lol Et qu'est-ce que tu vérifies Nous devons vérifier qu'i

  Cultures de couverture. Lequel dois-je utiliser pour s'adapter à ma rotation. Je m'appelle Michaela Schlathölter, je suis une
éleveur que PH. Petersen Saatzucht Lundsgaard. Une semence et un élevage privé et indépendant
compagnie. Situé dans la partie nord de l'Allemagne, à proximité
jusqu'à la frontière danoise. Nous sommes spécialistes de l'élevage résistant aux nématodes
couverts et autres cultures fourragères, céréales et les légumineuses. Nous faisons notre propre production, vente et commercialisation de semences
avec Saaten-Union. Et le tout avec des partenaires locaux. Le cycle commun des cultures de couverture commence
avec une récolte de la récolte principale en juillet commençant d'août. Jusqu'à la plantation de la culture principale suivante en
avril-mai de l'année prochaine. Cette période de cultures de couverture est une période moins productive
période de l'année, faute de soleil, longueur du jour et unités de chaleur. Nous parlerons de cinq façons clés d'améliorer
la 

  aller plus loin dans ce monde de terre
sciences de la santé et des sols et comprendre ce qui se passe avec votre
les plantes et le sol lui-même et aident créer le nouveau système alimentaire naturel et moderne tout le monde est le bienvenu à l'autre
la nature a toujours raison la vidéo d'aujourd'hui la vidéo va être sur le bon meilleur
technique de fourche large ainsi qu'une explication de ce qui est sans labour ce qui est
quelles sont les différences entre quand allons-nous faire chacun d'eux nous serons
parler beaucoup de la santé des sols dans général donc j'espère que vous apprendrez beaucoup
de plus de détails sur la façon dont le sol la structure se forme comment les différents
micro-organismes travaillent ensemble et certains plus de chimie impliquée dans le sol
processus de conversion alors tout d'abord définissons ce qu'est le labourage
donc labourer est-ce que tu sais que c'est un réel machine appelée motoculteur et la façon dont
cette machine fonctionne si elle tourne les 

   Salut les gens, c'est une belle journée pluvieuse en Californie et je suis vraiment ravie de pouvoir partager avec vous
comment faire une houe de recyclage. C'est un type spécial de houe que j'ai récemment
développé et qui nous permet de contrôler les mauvaises herbes dans une situation comme celle-ci en arrière-plan,
sans abîmer le plastique. J'ai développé ce nouveau design de houe que j'appelle le
Recyclez la houe. Et laissez-moi vous expliquer les trois parties qui
sont critiques. Donc, la première chose dont vous aurez besoin est un grand
morceau d'herbe. OK, donc c'est un morceau d'herbe. L'herbe s'appelle Arundo. Et c'est une mauvaise herbe très commune et envahissante ici
en Californie dans certaines régions. Et vous allez l'utiliser comme notre poignée. Ceux-ci font de grandes poignées. Pour notre lame ce que nous allons utiliser est un morceau de cet acier
cerclage. Il s'agit donc d'un cerclage en acier qui provient de
quincailleries. Vous n'avez pas à l'acheter. C'est trè

  Cette semaine sur Machines Pete
TV, matériel de travail du sol est sur le bloc d'enchères sur ce Missouri
vente, y compris une magnifique charrue International 720 restaurée.
De pièces à la quasi-perfection, nous apprendrons ce 501
Maître d'oeuvre. Et Pete demande, comment vieux doit être un tracteur,
être une antiquité VOS MACHINES EST UN INVESTISSEMENT SÉRIEUX – ET
AU CŒUR DE TOUTE AGRICULTURE OPÉRATION. CERTAINS L'APPELENT UN
LA PASSION. NOUS SOMMES MACHINES PETE TV, ET AUJOURD'HUI NOUS COUVRIRONS
TOUT AUX ENCHÈRES ROUNDUPS, AUX CLASSIQUES,
AUX DERNIÈRES TENDANCES ET LA TECHNOLOGIE. MACHINES PETE –
LE NOM LE PLUS FIABLE DE LA FERME ÉQUIPEMENT. Machinerie Pete merci
ces premiers sponsors pour leur soutien - SULLIVAN AUCTEURS.
LAISSEZ NOTRE ÉQUIPE DE PROFESSIONNELS VOUS MONTRER COMMENT FAIRE VOTRE
VENTE AUX ENCHÈRES RÉUSSIE. VISITE SULLIVAN AUCTIONEERS.COM. Hé
Bienvenue à Machinery Pete TV à Tarkio, Missouri, ce
la semaine. Et c'est un honneur pour moi être ici de la ferme
vente aux

  alors bienvenue à Kate's Ag et aujourd'hui je suis ici
avec mon voisin Steve et il va être nous montrant son héros qui est très très cool et
nous apprendre tout à ce sujet donc c'est super excitant salut comment vas-tu katie je vais bien merci euh ouais
Comme Kate l'a dit, c'est ce qu'on appelle un héraut lourd qui traversera le champ ce
un champ particulier sera le blé de printemps et aussi loin qu'un peu d'histoire sur la mienne
opération famille euh on continue euh 104 ans maintenant c'est incroyable euh je suis la quatrième génération
oh je suis aussi de la quatrième génération et il se trouve que vivre à l'endroit où en fait mon
arrière arrière grand-père incroyable ouais en fait de l'autre côté de la route pour où
nous vivons maintenant ce sont les arbres qui existent encore aujourd'hui c'est super intéressant et c'est incroyable et je
croient que l'histoire de chaque agriculteur devrait être racontée et c'est un genre totalement différent de partie de la
opération c'est tellem

  Un tracteur à deux roues ou un motoculteur sont
termes génériques compris aux États-Unis et dans certaines parties de l'Europe pour représenter un
tracteur à un essieu, qui est un tracteur à un essieu, automoteur et
automoteur, qui peut tirer et alimenter divers outils agricoles tels qu'un
une remorque, un cultivateur ou une herse, une charrue, ou divers semoirs et moissonneuses. La
l'opérateur marche généralement derrière ou monte l'outil remorqué. Similaire
termes sont appliqués par erreur au motoculteur ou motoculteur domestique ;
bien que ceux-ci puissent être à roues et/ou automoteurs, ils ne sont pas adaptés
pour les outils de remorquage. Un deux-roues tracteur se spécialise dans la traction de
nombreux types d'outils, alors que motobineuses spécialisées dans le sol
travail du sol avec leur creusage dédié outils. Cet article concerne les deux-roues
tracteurs par opposition à ces talles.
Confusion sur la définition La recherche a identifié un certain nombre de
termes utilisés po

  nous allons passer une merveilleuse journée et je suis
tellement excités que nous avons les cèdres et ils sont actuellement ensemencement nous avons deux semoirs dans l'un
champ et puis nous avons mon père avec son homme engagé et le tracteur ramassant des pierres et
c'est ce que nous faisons aujourd'hui donc l'ensemencement est très important et un élément clé
dans l'agriculture parce que si vous n'avez pas d'ensemencement et si vous ne le faites pas correctement, c'est le
le début de tout ce dans quoi vous plantez votre semence le sol et puis vous avez un merveilleux
récolte ou si cette partie ne va pas bien vous pourriez ne pas avoir la meilleure récolte donc c'est un
rôle très important dans l'agriculture et je vais vous emmener le long d'une journée typique d'ensemencement aujourd'hui
c'est dimanche donc nous ne pouvons pas acheter de semences aujourd'hui donc nous devra peut-être s'arrêter un peu plus tôt et ensuite
ramasser lundi demain alors allons-y c'est dan notre autre moi

In [17]:

df = pd.read_csv('sous_titrage.csv')
df.iloc[:,[1]]
len(df)

176